Friends/FriendModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const friendSchema = new mongoose.Schema({
+    firstName:{
+        type: String,
+        required: true
+    },
+    lastName:{
+        type: String,
+        required: true
+    },
+    age:{
+        type: Number,
+        required: true,
+        min: 1,
+        max: 120
+    },
+    createdOn:{
+        type: Date,
+        required: true,
+        default: Date.now
+    }
+});
+
+const friendModel = mongoose.model('Friends', friendSchema);
```

Comment:
Instead of having 'Friends' I would put Friend. Because the model takes in a third argument with if you leave it blank, it will default to the lowercase plural version of the name you give